.travis.yml

```diff
@@ -18,9 +19,9 @@ before_script:
   - sleep 5
 script:
   - npm run lint
-  - npm run unit
+  - if [[ $(node -v) =~ ^v4.* ]]; then npm run unit:harmony; else npm run unit; fi
```

Comment:
Curious if we could do like a `export isNode4=( $(node -v) =~ ^v4.*)` 
could then do `[[ isNode4 ]] && npm run unit:harmony || npm run unit`

would feel a little cleaner. :)